### PR TITLE
Lookup CSS properties case-insensitive

### PR DESCRIPTION
**Pull request by matthiasblaesing**
Closes: #4861

### CODE REVIEW

**Comment by matthiasblaesing** [comment]
Testbuild: https://doppel-helix.eu/NetBeans-dev-dev-6f2056b4929fc2089418b487defec1b4150efefa-release.zip

File: ide/css.editor/src/org/netbeans/modules/css/editor/module/main/BrowserSupportModule.java [hunk 1]
```diff
@@ -66,7 +66,12 @@ public Collection<String> getPropertyNames(FileObject file) {
 
     @Override
     public PropertyDefinition getPropertyDefinition(String propertyName) {
-        return parser.getVendorSpecificProperties().get(propertyName);
+        PropertyDefinition pd = parser.getVendorSpecificProperties().get(propertyName);
```
Comment (lbownik) [inline ide/css.editor/src/org/netbeans/modules/css/editor/module/main/BrowserSupportModule.java:69]: what is the result of this execution if propertyName is null? can it ever be null?

Comment (matthiasblaesing) [reply ide/css.editor/src/org/netbeans/modules/css/editor/module/main/BrowserSupportModule.java:69]: I don't understand what is the relevance of this question. If there was a problem before this change it is still there. This is not the point of the changeset.

To answer the question: `BrowserSpecificDefinitionParser#getVendorSpecificProperties` is backed by a `HashSet`. This allows `null` as key. The parser though will not yield `null` keys, so the `null` will not have a backing value, yielding `null` fro the `PropertyDefinition`. This in turn is caught in the new line 70, which explicitly checks for `null`.

Comment (lbownik) [reply ide/css.editor/src/org/netbeans/modules/css/editor/module/main/BrowserSupportModule.java:69]: ok - good 
 just saw a lack of null check and though there might be some potential NullPointer there.

**Pull request merged**